build.gradle

```diff
@@ -51,11 +51,12 @@ ext {
     gaVersion = '16.0.3'
     firebaseVersion = '16.0.3'
 
-    skeletoidVersion = '0.3.0'
+    skeletoidVersion = '0.4.0'
```

Comment:
Can we remove this and make use of project dependency instead of lib dependency?
Since if we use this we will always be using the previous version instead of the current one.